app/controllers/users_controller.rb

```diff
@@ -373,4 +374,24 @@ def acknowledge_cookies
     end
     true
   end
+
+  def anonymize
```

Comment:
Regarding your WRT workflows, how does this relate to anonymizing a `Person`?
I know that your existing anon logic is quite complex in terms of "does this person have posts on the forum, which comps is this person registered for, etc". These are all checks which _technically_ belong to `User`, not to Person. But you're running them all in the anonymizePerson step, so I want to make sure that this "simple" user anonymization here doesn't break any existing assumptions or workflows.